types/domtagger/test/domtagger.cjs.test.ts

```diff
@@ -0,0 +1,54 @@
+import domtagger = require('domtagger');
```

Comment:
I believe this supposed to be `const domtagger = require('domtagger');`

Reply:
**TypeScript**&nbsp;uses `import domtagger = require('domtagger');`, which&nbsp;gets&nbsp;transpiled to&nbsp;`const domtagger = require('domtagger');`.

Using&nbsp;`const domtagger = require('domtagger');` would&nbsp;infer the&nbsp;type of&nbsp;`domtagger` as&nbsp;`any`, and&nbsp;cause a&nbsp;build&nbsp;failure without&nbsp;a&nbsp;type&nbsp;definition for&nbsp;`require`.

---

**P.S.:** You&nbsp;need to&nbsp;select the&nbsp;**Approve** radio&nbsp;button to&nbsp;let&nbsp;@TypeScript-bot know&nbsp;that you’re&nbsp;approving&nbsp;this.